src/main/_data/sidebars/che_7_docs.yml

```diff
@@ -182,7 +182,9 @@ entries:
   - title: Troubleshooting for Che end users
     url: che-7/troubleshooting-for-che-end-users
     output: web
-
+  - title: Network Troubleshooting
```

Comment:
Also, don't use title case.

Reply:
Note this PR may come into conflict with https://github.com/eclipse/che-docs/pull/1431; I converted `Troubleshooting for Che end users` into a folder: https://github.com/eclipse/che-docs/pull/1431/commits/96c50d7dd3a365221ba702d46ba5276a7037330e#diff-0a2cd2b5d2893e3f8ea010950e61b51bR182-R196

Reply:
I got Network Troubleshooting as a first idea from original issue. I am totaly agre - it is better to move this section under "Troubleshooting for Che end users.
I waited for merge the long term [PR](https://github.com/eclipse/che-docs/pull/1431)  to avoid conflict.